docs/src/api.md

```diff
@@ -65,7 +65,7 @@ StatsBase.deviance
 GLM.dispersion
 GLM.ftest
 GLM.installbeta!
-GLM.issubmodel
+StatsModels.isnested
```

Comment:
should the position be changed to maintained alphabetical (when including the module name) order?